health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/ConsoleToolTip.js

```
@@ -1,17 +1,13 @@
 import React from "react";
 import { InfoOutline } from "@egovernments/digit-ui-svg-components";
 import { TooltipWrapper } from "@egovernments/digit-ui-components";
 
-const ConsoleTooltip = ({toolTipContent,placement="bottom-end",iconFill="#787878"}) => {
+const ConsoleTooltip = ({ className, toolTipContent, placement = "bottom-end", iconFill = "#787878" }) => {
   return (
-    <span className="icon-wrapper">
-
-    <TooltipWrapper content={toolTipContent} placement={placement} children={<InfoOutline fill={iconFill} width={"20px"} height={"20px"} />} />
+    <span className={`icon-wrapper ${className ? className : ""}`}>
+      <TooltipWrapper content={toolTipContent} placement={placement} children={<InfoOutline fill={iconFill} width={"20px"} height={"20px"} />} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)